diff --git a/cmake/LibraryConfigurations.cmake b/cmake/LibraryConfigurations.cmake
index 96abeea9..2ce4e53f 100644
--- a/cmake/LibraryConfigurations.cmake
+++ b/cmake/LibraryConfigurations.cmake
@@ -27,6 +27,12 @@ if (${PLATFORM} MATCHES "Desktop")
         add_definitions(-D_CRT_SECURE_NO_WARNINGS)
         find_package(OpenGL QUIET)
         set(LIBS_PRIVATE ${OPENGL_LIBRARIES} winmm)
+    elseif("${CMAKE_SYSTEM_NAME}" MATCHES "QNX")
+           set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
+           find_library(GLESV2 GLESv2)
+           find_library(EGL EGL)
+           set(LIBS_PUBLIC m)
+           set(LIBS_PRIVATE ${GLESV2} ${EGL} atomic pthread dl)
     elseif (UNIX)
         find_library(pthread NAMES pthread)
         find_package(OpenGL QUIET)
